from src.ejercicio5 import verificar_contrase単a
import pytest

def test_contrase単a_excepcion():
    with pytest.raises(NameError) as excinfo:
        verificar_contrase単a("contrase単a_incorrecta")
    
    assert str(excinfo.value) == "Incorrect Password!!"